### PR TITLE
Specify size (in bytes) of log entry components

### PR DESCRIPTION
**Pull request by SarahPythonista**

### CODE REVIEW

**Comment by SarahPythonista** [comment]
@a-dma Could you review this? Thanks!

File: content/YubiHSM2/Commands/Get_Logs.adoc [hunk 1]
```diff
@@ -45,10 +45,10 @@ audit enforce is set) (2 bytes)
...
 N := Number of elements in the list (1 byte)
 
-E~i~ := Generic log entry composed of the command number, command ID,
-command length, the Session originating the command, the target key affected
-by the command, a secondary key if the command affected more than one key,
-the result byte, the systick when the command was processed and a digest. If the
+E~i~ := Generic log entry composed of the command number (two bytes), command ID (one byte),
+command length (two bytes), the Session originating the command (two bytes), the target key affected
+by the command (two bytes), a secondary key if the command affected more than one key (two bytes),
+the result byte, the systick when the command was processed (4 bytes) and a digest (16 bytes). If the
```
Comment (a-dma) [inline content/YubiHSM2/Commands/Get_Logs.adoc]: how about adding the (obvious) length after `result byte` as well? For consistency's sake :)

Something like `the result of the command (one byte)` will probably make it less obtuse.

Comment (SarahPythonista) [reply content/YubiHSM2/Commands/Get_Logs.adoc]: Sounds good!

**Comment by a-dma** [comment]
Thanks

**Pull request merged**